config/settings/testing.py

```diff
@@ -21,6 +21,9 @@
         'USER': 'root',
         'PASSWORD': '',
         'NAME': 'sortinghat_db',
+        'OPTIONS': {
+            'charset': 'utf8mb4'
```

Comment:
when testing the PR, I'm getting the following error: 
```
ERROR: test_duplicate_identities_with_truncated_values (tests.test_api.TestAddIdentity)
Check if the same identity with truncated values is not inserted twice
...
mysql_exceptions.DataError: (1406, "Data too long for column 'email' at row 1")
```

To fix it, I would add to `OPTIONS` the following entry to to swtich MySQL mode to not STRICT
`'sql_mode': 'NO_AUTO_CREATE_USER,NO_ENGINE_SUBSTITUTION'`. What do you think?

Reply:
I got that error but I disabled it doing something similar in my MySQL config. This makes more sense. I will add some other modes that I think they should still there.

Reply:
I've just uploaded some changes that fix it.